### PR TITLE
164796901 Implementing Article search functionality

### PR DESCRIPTION
**Pull request by toluola**
#### Description
- A user should be able to search for an article using the title of the title and keyword

#### Type of change
-  New feature (a non-breaking change which adds functionality)

#### How Has This Been Tested?
- Integration

#### Checklist:
- [x] my code follows style guidelines for this project

#### PT
- [#164796901](https://www.pivotaltracker.com/story/show/164796901)

#### Postman Screenshot
Screenshot 2019-04-18 at 4 22 09 PM

### CODE REVIEW

File: src/migrations/search.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      ).then(() => queryInterface.sequelize.query(`
+                CREATE INDEX ${table}_search ON ${table} USING gin(${vectorName});
+              `, { transaction: t })
+      ).then(() => queryInterface.sequelize.query(`
+                CREATE TRIGGER ${table}_vector_update
+                BEFORE INSERT OR UPDATE ON ${table}
+                FOR EACH ROW EXECUTE PROCEDURE tsvector_update_trigger(${vectorName}, 'pg_catalog.english', ${searchObjects[table].join(', ')});
+              `, { transaction: t })
+      )
+      .error(console.log)
```
Comment (Temmyogunbo) [inline src/migrations/search.js]: why the console?

**Review by micah-akpan — COMMENTED** [review]
@toluola can you squash your commits into 1 - 5 commits?

**Comment by toluola** [comment]
> @toluola can you squash your commits into 1 - 5 commits?

@micah-akpan thanks for the feedback, it's been agreed that the TTL will squash all commits before merging.

**Review by chikeozulumba — COMMENTED** [review]
LGTM, Nice Job done. @toluola

**Comment by micah-akpan** [comment]
LGTM

**Comment by ssanusi** [comment]
LGTM @toluola nice work adapting to new method so fast

**Comment by Simplemart17** [comment]
LGTM @toluola

**Review by Simplemart17 — COMMENTED** [review]
LGTM @toluola

**Review by Temmyogunbo — APPROVED** [review]

**Pull request merged**